src/prayer/AnswerPrayerForm/AnswerPrayerFormConnected.js

```diff
@@ -1,11 +1,30 @@
 import React from 'react';
+import { View } from 'react-native';
 import { Query, Mutation } from 'react-apollo';
+import { BodyText, styled } from '@apollosproject/ui-kit';
 import getUserProfile from '../../tabs/connect/UserAvatarHeader/getUserProfile';
 import GET_PRAYER_FEED from '../data/queries/getPrayerFeed';
 import ANSWER_PRAYER from '../data/mutations/answerPrayer';
+import INTERACT_WITH_PRAYER from '../data/mutations/interactWithPrayer';
+import ActionComponent from '../ActionComponent';
 import AnswerPrayerForm from './AnswerPrayerForm';
 
+const OptionMenu = styled(({ theme }) => ({
+  alignSelf: 'center',
+  padding: theme.sizing.baseUnit,
+}))(View);
+
+const OptionMenuText = styled(({ theme }) => ({
+  color: theme.colors.text.tertiary,
+}))(BodyText);
+
 class AnswerPrayerFormConnected extends React.Component {
+  prayer = {
+    id: this.props.navigation.getParam('prayerId', ''),
+    text: this.props.navigation.getParam('prayerText', ''),
+    answer: this.props.navigation.getParam('prayerAnswer', ''),
```

Comment:
couldn't we pass these in as one combined nav param?

Reply:
I'm not sure what you mean.

Reply:
```js
prayer = this.props.navigation.getParam('prayer', {id: "", text: "", answer: ""})
```